src/emmetHelper.ts

```diff
@@ -1048,6 +1067,10 @@ function resetSettingsFromFile() {
 	variablesFromFile = {};
 }
 
+function isValidExtensionsPath(path: string){
```

Comment:
Not sure what this function is for.
Also, I recommend running `Format Document` on all these files.

Reply:
Oh I forgot to delete the unused function. I will do the format before next push.